src/Type/Constant/ConstantArrayType.php

```diff
@@ -386,6 +390,13 @@ public function findTypeAndMethodName(): ?ConstantArrayTypeAndMethod
 			if ($this->isOptionalKey(0) || $this->isOptionalKey(1)) {
 				$has = $has->and(TrinaryLogic::createMaybe());
 			}
+			if (
+				$isClassString &&
+				$has->yes() &&
+				!$type->getMethod($method->getValue(), new OutOfClassScope())->isStatic()
```

Comment:
I like having `&&` at the beginning of the lines more, like here: https://github.com/phpstan/phpstan-src/blob/9beb27ea1c0b695a32e64ff94c9c65606f190b0c/src/Analyser/NodeScopeResolver.php#L358-L364

Reply:
Thanks, I agree – I will definitely be stealing this approach